packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.html

```diff
@@ -2,14 +2,12 @@
   <host>
     <article class="card">
       <header class="slds-grid slds-p-horizontal_x-small slds-has-flexi-truncate">
-        <div class="slds-media__body">
-          <div class="slds-grid card__header">
+        <div class="slds-media__body card_focusable-header" onclick={handleHeaderClick} onkeydown={handleHeaderKeyDown} role="button" tabindex="0">
```

Comment:
If `<slot name="actions"></slot>` contains interactive elements, then I recommend making sure there's no parent interactive elements (See [Deque `nested-interactive`](https://dequeuniversity.com/rules/axe/4.2/nested-interactive)).

One solution If you want the header to still be clickable is to make it clickable but not indicate to screen readers that it is a button, and instead make a new button as a child which has the same behavior.

Reply:
Thank you for pointing this out! however, we are not passing interactive elements in the `actions` slot. We passing a lightning-button component but we are adding the attribute aria-hidden="true"    to make it non-interactive.